### DISCUSSION TITLE
[Expo Image] Sometimes an error occurs when I use placeholder with blurhash(Only Android)

### DISCUSSION THREAD

**Discussion by WinKoKoNaing** [expo/expo#21286] [Archived: Questions]
```
const blurhash =
  "|rF?hV%2WCj[ayj[a|j[az_NaeWBj@ayfRayfQfQM{M|azj[azf6fQfQfQIpWXofj[ayj[j[fQayWCoeoeaya}j[ayfQa{oLj?j[WVj[ayayj[fQoff7azayj[ayj[j[ayofayayayj[fQj[ayayj[ayfjj[j[ayjuayj[";

 <FlashList
      ListHeaderComponent={renderHeader}
      data={listings}
      numColumns={2}
      onEndReached={onPressMore}
      onEndReachedThreshold={1}
      showsVerticalScrollIndicator={false}
      renderItem={renderItem}
      keyExtractor={(item, index) => item._id?.toString() + index}
      estimatedItemSize={180}
      estimatedListSize={{ width: 160, height: 203 }}
      ListFooterComponent={() => (
        <>{!isReachingEnd && isLoadingMore && <Loading />}</>
      )}
    />

const renderItem = ({ item, index }) => (
    <Image
  style={{
    width: 149.39,
    height: 149.39,
  }}
  source={item?.photos?.[0]}
  placeholder={blurhash}
  contentFit="cover"
  transition={1000}
 
/>;
  );
 
```

The above is some part of my code. does it occur in glide?

Environment
Expo SDK 48, expo-image RC.1, flashlist 1.4.0

The error is 
`ERROR  Your app just crashed. See the error below.
java.lang.RuntimeException: Canvas: trying to use a recycled bitmap android.graphics.Bitmap@189434a
  android.graphics.BaseCanvas.throwIfCannotDraw(BaseCanvas.java:74)
  android.graphics.MiuiCanvas.throwIfCannotDraw(MiuiCanvas.java:358)
  android.graphics.RecordingCanvas.throwIfCannotDraw(RecordingCanvas.java:263)
  android.graphics.BaseRecordingCanvas.drawBitmap(BaseRecordingCanvas.java:94)
  android.graphics.drawable.BitmapDrawable.draw(BitmapDrawable.java:549)
  android.widget.ImageView.onDraw(ImageView.java:1480)
  expo.modules.image.ExpoImageView.onDraw(ExpoImageView.kt:186)
  android.view.View.draw(View.java:23001)
  expo.modules.image.ExpoImageView.draw(ExpoImageView.kt:182)
  android.view.View.updateDisplayListIfDirty(View.java:21869)
  android.view.View.draw(View.java:22727)
  android.view.ViewGroup.drawChild(ViewGroup.java:4542)
  android.view.ViewGroup.dispatchDraw(ViewGroup.java:4300)
  android.view.View.updateDisplayListIfDirty(View.java:21858)
  android.view.View.draw(View.java:22727)
  android.view.ViewGroup.drawChild(ViewGroup.java:4542)
  android.view.ViewGroup.dispatchDraw(ViewGroup.java:4300)
  android.view.View.updateDisplayListIfDirty(View.java:21858)
  android.view.View.draw(View.java:22727)
  android.view.ViewGroup.drawChild(ViewGroup.java:4542)
  com.facebook.react.views.view.ReactViewGroup.drawChild(ReactViewGroup.java:836)
  android.view.ViewGroup.dispatchDraw(ViewGroup.java:4300)
  com.facebook.react.views.view.ReactViewGroup.dispatchDraw(ReactViewGroup.java:809)
  android.view.View.draw(View.java:23005)
  android.view.View.updateDisplayListIfDirty(View.java:21869)
  android.view.View.draw(View.java:22727)
  android.view.ViewGroup.drawChild(ViewGroup.java:4542)
  com.facebook.react.views.view.ReactViewGroup.drawChild(ReactViewGroup.java:836)
  android.view.ViewGroup.dispatchDraw(ViewGroup.java:4300)
  com.facebook.react.views.view.ReactViewGroup.dispatchDraw(ReactViewGroup.java:809)
  android.view.View.draw(View.java:23005)
  android.view.View.updateDisplayListIfDirty(View.java:21869)
  android.view.View.draw(View.java:22727).....`

**Comment by a-eid** [comment]
@WinKoKoNaing seems to be an issue rather than a discussion, also a stable release was released recently, you should give it a try.

**Reply by WinKoKoNaing** [reply]
Yes, now it doesn't occur, when I added recyclingKey in image

**Reply by WinKoKoNaing** [reply]
today when I released the app, it happens again and i don't know why
